library/kotlin/src/io/envoyproxy/envoymobile/stats/CounterImpl.kt

```diff
@@ -15,6 +15,7 @@ internal class CounterImpl : Counter {
     this.elements = elements
   }
 
+  /// TODO: raise error to platform if the operation is not successful.
```

Comment:
```suggestion
  /// TODO: Potentially signal error to platform if the operation is not successful.
```

It's not altogether certain to me that we should raise, though it is an option.